docs/logging.md

```diff
@@ -0,0 +1,48 @@
+# Logs from execution
+
+## Log location
+
+Default location for logs is under `/tmp` folder.
+User can change it by providing own location via config file passed via `--ocsci-conf`:
+
+```yaml
+---
+RUN:
+    log_dir: /my/custom/log/location
+```
+
+In the log_dir defined above, the run-ci will create new folder per execution
+named in format of: `ocs-ci-logs-$RUN_TYPE-$JENKINS_BUILD_ID-$RUN_ID`.
```

Comment:
I think that the format will be little bit shifted - first is the `RUN_ID` and then the additional info:
```
f"ocs-ci-logs-{config.RUN['run_id']}{additional_info}",
```
Also the `JENKINS_BUILD_ID` is valid only for Jenkins execution, so it might be worth to mention it that for non-jenkins executions, this will be empty.